taxcalc/functions.py

```diff
@@ -1525,7 +1519,7 @@ def AddCTC(_nctcr, _precrd, _earned, c07220, e00200, e82882, e30100, _sey,
     # Part II of 2005 form 8812
 
     if _nctcr >= ACTC_ChildNum and c82890 < c82935:
-        c82900 = 0.0765 * min(SS_Earnings_c, c82880)
+        c82900 = 0.0765 * min(min(SS_Earnings_c, c82880), e00200)
```

Comment:
The `0.0765` seems to be the FICA rate, but I'm still a little bit confused by the changes made here.